services/motion/builtin/builtin.go

```diff
@@ -611,7 +607,7 @@ func (ms *builtIn) execute(ctx context.Context, trajectory motionplan.Trajectory
 				if err != nil {
 					return err
 				}
-				if !checkSameInputs(inputs, curr, epsilon) {
+				if referenceframe.InputsLinfDistance(curr, inputs) > epsilon {
```

Comment:
function for an if statement could just be an if statement. 

and i even wrote a fancy test for it too